src/oscillator.js

```diff
@@ -213,6 +213,12 @@ define(function (require) {
   // these are now the same thing
   p5.Oscillator.prototype.fade =   p5.Oscillator.prototype.amp;
 
+  /**
+   * Returns the value of output gain 
+   *
+   *  @method  getAmp
+   *  @for p5.Oscillator
+   */ 
```

Comment:
Should probably add a `@returns {number}` here, the description could be something like `Amplitude value between 0.0 and 1.0`